packages/buildpacks/src/commands/buildpacks/add.ts

```diff
@@ -0,0 +1,42 @@
+import {Command, flags as Flags} from '@heroku-cli/command'
+
+import {BuildpackCommand} from '../../buildpacks'
+
+export default class Add extends Command {
+  static description: 'add new app buildpack, inserting into list of buildpacks if necessary'
+  static flags = {
+    app: Flags.app({required: true}),
+    remote: Flags.remote(),
+    index: Flags.integer({
+      name: 'index',
+      description: 'the 1-based index of the URL in the list of URLs',
+      char: 'i'
+    })
+  }
+  static args = [
+    {name: 'buildpack', required: true}
+  ]
+
+  async run() {
+    let {args, flags} = this.parse(Add)
+    let buildpackCommand = new BuildpackCommand(this.heroku)
+
+    if (flags.index !== undefined) {
+      buildpackCommand.validateIndex(flags.index)
+    }
+
+    let buildpacks = await buildpackCommand.fetch(flags.app)
+    await buildpackCommand.validateUrlNotSet(buildpacks, args.buildpack)
+
+    let spliceIndex: number
+    if (flags.index === undefined) {
```

Comment:
out of scope probably, but I would like it at some point if this used inquirer to prompt so you could select where you wanted to insert the buildpack visually

Reply:
@jdxcode sounds good to me. don't you own this after we merge it in? :D

Reply:
yeah I'm basically talking to myself